tests/conftest.py

```diff
@@ -164,6 +164,6 @@ def schema():
             },
         }, {
             '$ref': 'http://inveniosoftware.org/schemas/'
-                    'oaiserver/internal-v1.0.0.json',
+                    'oaiserver/internal-v1.1.0.json',
```

Comment:
I think it's ok not to make a test since we're not changing the schema in anyway